src/environments/environment.ts

```diff
@@ -5,6 +5,7 @@
 // This file can be replaced during build by using the `fileReplacements` array.
 // `ng build --prod` replaces `environment.ts` with `environment.prod.ts`.
 // The list of file replacements can be found in `angular.json`.
+declare var require: any;

```

Comment:
What is the require variable used for?

Reply:
It resolves [this error](https://javascript.plainenglish.io/typescript-cannot-find-name-require-8e327dde6363) that showed up after the fresh npm install.